SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetBitReader.h

```diff
@@ -16,16 +16,16 @@ class SPATIALGDK_API FSpatialNetBitReader : public FNetBitReader
 public:
 	FSpatialNetBitReader(USpatialPackageMapClient* InPackageMap, uint8* Source, int64 CountBits, TSet<FUnrealObjectRef>& InDynamicRefs, TSet<FUnrealObjectRef>& InUnresolvedRefs);
 
-	using FArchive::operator<<; // For visibility of the overloads we don't override
+	~FSpatialNetBitReader();
 
-	virtual FArchive& operator<<(UObject*& Value) override;
+	using FArchive::operator<<; // For visibility of the overloads we don't override
 
-	virtual FArchive& operator<<(struct FWeakObjectPtr& Value) override;
+	static UObject* ReadObject(FArchive& Archive, USpatialPackageMapClient* PackageMap, bool& bUnresolved);
 
-	UObject* ReadObject(bool& bUnresolved);
+	FArchive& operator<<(FWeakObjectPtr& Value);
```

Comment:
Prefer keeping the explicit override (to make it more obvious if this changes in the future engine versions)